### PR TITLE
New Ember.js Cheat sheet

### PR DESCRIPTION
**Pull request by navneetzz**
emberjs
**What does your Instant Answer do?**
My Instant answer provide reference to basic syntax of ember.js.

---
https://duck.co/ia/view/emberjs_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Emberjs Cheat Sheet](https://duck.co/ia/view/emberjs_cheat_sheet)

**Description**: A quick reference to ember.js syntax

**Example Query**: [emberjs cheat sheet](https://beta.duckduckgo.com/?q=emberjs cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: Ember.js

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/emberjs_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/emberjs_cheat_sheet)*

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=emberjs cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/emberjs_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/emberjs.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
+    "id": "emberjs_cheat_sheet",
+    "name": "Ember.js",
+    "description": "A quick reference for ember.js, an open-source JavaScript application framework",
+    "metadata": {
+        "sourceName": "Ember.js",
+        "sourceUrl": "https://guides.emberjs.com/v2.3.0/"
+    },
+    "aliases": [
+    "ember.js"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/emberjs.json]: Spacing is a bit off here.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=emberjs cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/emberjs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=emberjs cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/emberjs_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/emberjs.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
+{
+    "id": "emberjs_cheat_sheet",
+    "name": "Ember.js",
+    "description": "A quick reference for ember.js, an open-source JavaScript application framework",
+    "metadata": {
+        "sourceName": "Ember.js",
+        "sourceUrl": "https://guides.emberjs.com/v2.3.0/"
+    },
+    "aliases": [ "ember.js" ],
+    "template_type": "reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/emberjs.json]: It looks like these are all code snippets? Let's use `terminal` or `code`

**Comment by moollaza** [comment]
@navneetzz thanks! This LGTM :+1:

**Comment by moollaza** [comment]
@navneetzz thanks again for creating the Ember Cheat Sheet! You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Ember.js.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to Ember.js

OR

- Test live Ember.js Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Pull request merged**